model/database/registry.yaml

```diff
@@ -106,10 +106,10 @@ groups:
         type: string
         stability: release_candidate
         brief: >
-          Low cardinality representation of a database query text.
-        note: >
-          `db.query.summary` provides static summary of the query text. It describes
-          a class of database queries and is useful as a grouping key,
+          Low cardinality summary of a database query.
+        note: |
+          `db.query.summary` provides static summary of the query.
```

Comment:
(it was there previously, but I think the word possibly raises more questions than it answers)
```suggestion
          `db.query.summary` provides summary of the query.
```

Reply:
or could probably just remove this line since redundant with brief?